packages/client/src/subscribe/OrderMessages.ts

```diff
@@ -146,8 +146,8 @@ export class OrderMessages<T> {
         }
     }
 
-    transform(): (src: AsyncGenerator<StreamMessage<T>, any, unknown>) => AsyncGenerator<StreamMessage<T>, void, unknown> {
-        return async function* Transform(this: OrderMessages<T>, src: AsyncGenerator<StreamMessage<T>>) {
+    transform(): (src: AsyncGenerator<StreamMessage, any, unknown>) => AsyncGenerator<StreamMessage, void, unknown> {
```

Comment:
Could simplify AsyncGenerator generics to be just `AsyncGenerator<StreamMessage>` (in this PR on in a separate PR)

Reply:
Fixed in a5e9c88e854ce597f323272888a637a5150aa834